lib/web/fotorama/fotorama.js

```diff
@@ -1140,7 +1140,12 @@ fotoramaVersion = '4.6.4';
 
     function addEvent(el, e, fn, bool) {
         if (!e) return;
-        el.addEventListener ? el.addEventListener(e, fn, !!bool) : el.attachEvent('on' + e, fn);
+        const options = {
+            get passive() {
+                return e === 'wheel';
+            }
+        }
+        el.addEventListener ? el.addEventListener(e, fn, options) : el.attachEvent('on' + e, fn);
```

Comment:
Can we drop support for unsupported Internet Explorer v8 yet?
https://caniuse.com/addeventlistener
https://experienceleague.adobe.com/docs/commerce-operations/installation-guide/system-requirements.html#supported-browsers
```suggestion
        el.addEventListener(e, fn, options);
```

Reply:
sure let's try it

Reply:
updated code can you help for reviews